src/db/ClimbSchema.ts

```diff
@@ -59,6 +59,28 @@ const MetadataSchema = new Schema<IClimbMetadata>({
   }
 }, { _id: false })
 
+const PitchSchema = new mongoose.Schema({
+  _id: {
+    type: 'object',
+    value: { type: 'Buffer' },
+    default: () => muuid.v4()
+  },
+  uuid: {
+    type: 'string',
+    default: function () { return this._id.toString() }
+  },
+  parentId: { type: String, required: true },
+  number: { type: Number, required: true },
+  grades: { type: mongoose.Schema.Types.Mixed },
+  type: { type: mongoose.Schema.Types.Mixed },
+  length: { type: Number },
+  boltsCount: { type: Number },
+  description: { type: String }
+}, {
+  _id: true,
```

Comment:
we should remove _id: true since we will be providing our own UUID.  `_id: true` means Mongo will auto-generate _id for us in ObjectID format (not UUID).

Reply:
@vnugent I have incorporated all changes and would be happy to get your re-review, especially on lines 72-81, since I am not sure if I have interpreted your suggestions correctly: https://github.com/OpenBeta/openbeta-graphql/pull/350/files#diff-9b4028942ce1d106899ff51138cc6ee7cc75ea36110257394c51424643261967R72-R81